pages/reference-smart-account/transactions/execTransaction.mdx

```diff
@@ -0,0 +1,292 @@
+                0,
+                0,
+                0,
+                address(0),
+                payable(0),
+                "0x..."
+            );
+        }
+    }
+    ```
+  </Tabs.Tab>
+</Tabs>
+
+{/* <!-- vale on --> */}
+
+## Returns
+
+- **Type:** `bool`
+
+Boolean indicating transaction's success.
+
+## Parameters
+
+### `to`
+
+- **Type:** `address`
+
+Destination address of Safe transaction.
+
+```solidity focus=2
+(ISafe safe).execTransaction(
+    "0x...",
+    0,
+    "0x...",
+    Enum.Operation.Call,
+    0,
+    0,
+    0,
+    address(0),
+    payable(0),
+    "0x..."
+);
+```
+
+### `value`
+
+- **Type:** `uint256`
+
+Ether value of Safe transaction.
+
+```solidity focus=3
+(ISafe safe).execTransaction(
+    "0x...",
+    0,
+    "0x...",
+    Enum.Operation.Call,
+    0,
+    0,
+    0,
+    address(0),
+    payable(0),
+    "0x..."
+);
+```
+
+### `data`
+
+- **Type:** `bytes`
+
+Data payload of Safe transaction.
+
+```solidity focus=4
+(ISafe safe).execTransaction(
+    "0x...",
+    0,
+    "0x...",
+    Enum.Operation.Call,
+    0,
+    0,
+    0,
+    address(0),
+    payable(0),
+    "0x..."
+);
+```
+
+### `operation`
+
+- **Type:** `Enum.Operation`
+
+Operation type of Safe transaction.
+
+```solidity focus=5
+(ISafe safe).execTransaction(
+    "0x...",
+    0,
+    "0x...",
+    Enum.Operation.Call,
+    0,
+    0,
+    0,
+    address(0),
+    payable(0),
+    "0x..."
+);
+```
+
+### `safeTxGas`
+
+- **Type:** `uint256`
+
+Gas that should be used for the Safe transaction.
+
+```solidity focus=6
+(ISafe safe).execTransaction(
+    "0x...",
+    0,
+    "0x...",
+    Enum.Operation.Call,
+    0,
+    0,
+    0,
+    address(0),
+    payable(0),
+    "0x..."
+);
+```
+
+### `baseGas`
+
+- **Type:** `uint256`
+
+Gas costs that are independent of the transaction execution (for example base transaction fee, signature check, payment of the refund).
+
+```solidity focus=7
+(ISafe safe).execTransaction(
+    "0x...",
+    0,
+    "0x...",
+    Enum.Operation.Call,
+    0,
+    0,
+    0,
+    address(0),
+    payable(0),
+    "0x..."
+);
+```
+
+### `gasPrice`
+
+- **Type:** `uint256`
+
+Gas price that should be used for the payment calculation.
+
+```solidity focus=8
+(ISafe safe).execTransaction(
+    "0x...",
+    0,
+    "0x...",
+    Enum.Operation.Call,
+    0,
+    0,
+    0,
+    address(0),
+    payable(0),
+    "0x..."
+);
+```
+
+### `gasToken`
+
+- **Type:** `address`
+
+Token address (or 0 if ETH) that is used for the payment.
+
+```solidity focus=9
+(ISafe safe).execTransaction(
+    "0x...",
+    0,
+    "0x...",
+    Enum.Operation.Call,
+    0,
+    0,
+    0,
+    address(0),
+    payable(0),
+    "0x..."
+);
+```
+
+### `refundReceiver`
+
+- **Type:** `address payable`
+
+Address of receiver of gas payment (or 0 if tx.origin).
+
+```solidity focus=10
+(ISafe safe).execTransaction(
+    "0x...",
+    0,
+    "0x...",
+    Enum.Operation.Call,
+    0,
+    0,
+    0,
+    address(0),
```

Comment:
why are you using `0x...` for the first parameter and `adddess(0)` for this one? This also happens in other places. Should we use `address(0)` everywhere?